rbac/management/inventory_replicator/inventory_api_replicator.py

```
@@ -0,0 +1,353 @@
+        self.reason = "unknown"
+        self.metadata = {}
+
+        try:
+            status = rpc_status.from_call(error)
+            if status is not None and status.details:
+                detail = status.details[0]
+                info = error_details_pb2.ErrorInfo()
+                detail.Unpack(info)
+                self.reason = info.reason
+                self.metadata = json.loads(str(info.metadata).replace("'", '"'))
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)